src/libs/API.js

```diff
@@ -187,6 +187,7 @@ function getAuthToken() {
  * @param {String} parameters.partnerUserID
  * @param {String} parameters.partnerUserSecret
  * @param {String} [parameters.twoFactorAuthCode]
+ * @param {String} [parameters.email]
```

Comment:
NAB: Why are some of these in brackets and others are not?

Reply:
Brackets mean the parameter is optional.

Reply:
TIL, thanks!